.github/workflows/regression_tests.yml

```diff
@@ -26,6 +26,14 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt
           pip freeze | egrep 'iree|tensorflow'
+          python -m pip install flake8 pytest
+          
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude lit.cfg.py
+          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude lit.cfg.py
```

Comment:
I don't have a strong opinion on this particular lint check, since I don't contribute to this repo often. However, my general impression is that this repo is less stringent with style, testing, code review, etc. so these checks may be a little heavy to introduce.